From c406ca41b3d28fb0a9b418fc03a8c5c393f6bc8c Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Mon, 27 Sep 2021 23:09:50 -0300
Subject: [PATCH 115/322] [FIO toup] imx8mm_evk: increase
 CONFIG_SYS_SPL_MALLOC_SIZE to 16MB

commit 03f1f78a9b44 ("spl: fit: Prefer a malloc()'d buffer for loading
images")' changed the way buffer allocation worked for SPL to a more
flexible method.

When loading a complex u-boot FIT image (e.g. with TF-A, OP-TEE and
U-Boot proper) it is required larger buffer, since such FIT can easily
go over 10MB.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 include/configs/imx8mm_evk.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx8mm_evk.h b/include/configs/imx8mm_evk.h
index 35a42c49bf0..214288f96de 100644
--- a/include/configs/imx8mm_evk.h
+++ b/include/configs/imx8mm_evk.h
@@ -34,7 +34,7 @@
 #define CONFIG_SPL_BSS_START_ADDR	0x910000
 #define CONFIG_SPL_BSS_MAX_SIZE		SZ_8K	/* 8 KB */
 #define CONFIG_SYS_SPL_MALLOC_START	0x42200000
-#define CONFIG_SYS_SPL_MALLOC_SIZE	SZ_512K	/* 512 KB */
+#define CONFIG_SYS_SPL_MALLOC_SIZE	SZ_16M
 
 /* malloc f used before GD_FLG_FULL_MALLOC_INIT set */
 #define CONFIG_MALLOC_F_ADDR		0x912000
-- 
2.34.1

